packages/twenty-front/src/modules/object-metadata/utils/validateMetadataLabel.ts

```diff
@@ -1,4 +1,4 @@
-const metadataLabelValidationPattern = /^[a-zA-Z][a-zA-Z0-9 ]*$/;
+const metadataLabelValidationPattern = /^[^0-9].*$/;
```

Comment:
@ijreilly with this regex we don't exclude special characters which will also break in the BE for field creations